launcher/StringUtils.cpp

```diff
@@ -74,3 +76,16 @@ int StringUtils::naturalCompare(const QString& s1, const QString& s2, Qt::CaseSe
     // The two strings are the same (02 == 2) so fall back to the normal sort
     return QString::compare(s1, s2, cs);
 }
+
+QString StringUtils::getRandomAlphaNumeric(const int length)
```

Comment:
This would add "{}" to the beginning and end which would not be ideal for passing around as windows filename characters. those could be stripped like it did elsewhere. but a UUID seems *really* overkill for getting a temporary unique named pipe.

Reply:
For the braces, you can control the behavior of `toString` with [`this handy enum`](https://doc.qt.io/qt-6/quuid.html#StringFormat-enum).

And well, the purpose of UUID is to have a unique thingy for you to use, so why would it be overkill to use it for what it's intended? xD

Reply:
I suppose it doesn't matter *that* much, and it's a much stronger guarantee of being unique (not that it needs to *be* that strong)
I wish I had read the docs more thoroughly, that enum is nice.